kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/listeners/spec.kt

```diff
@@ -46,6 +46,17 @@ interface AfterSpecListener : Listener {
    suspend fun afterSpec(spec: Spec): Unit = Unit
 }
 
+interface SkipSpecListner : Listener {
+
+   /**
+    * Called once per [Spec] iff all tests in the spec are inactive.
+    *
+    * @param spec the [Spec] instance.
+    * @param results a map of each test case mapped to its skipped result.
+    */
+   suspend fun skipSpec(spec: Spec, results: Map<TestCase, TestResult>): Unit = Unit
```

Comment:
skipSpec sounds like a command, when this is reactive.
specSkipped ?

We also use the term disabled/ignored everywhere else, so maybe avoid the term skip completely.

specIgnored ?